### ISSUE TITLE
Create JSON for Section 3G

### ISSUE THREAD

**Issue by tadhg-ohiggins** [18F/cms-carts-seds#356]
Produce the JSON that will represent section 3g.

A/C:
- [ ] JSON file exists with content for every form element in section 3g.

Related to #339. Blocks #468 

NOTE: Claire - please work with Tony on CL to get him speed

**Comment by erinzstrenio** [comment]
Once the JSON is created, I'll add the google doc to the shared CL folder 

- [ ] @erinzstrenio moves Google doc into CL folder

**Comment by jgillis1** [comment]
ROLLOVER NOTE: The content of 3G has changed several times during Sprint I, the most recent being in the evening on 9/23/2020. As of 8:52 AM 9/24/2020, Andrew needs final confirmation that he is interpreting and implementing the content changes made correctly.

Issues Raised:
- help_drawer

- checkbox about whether or not the State collects data via age ranges which then changes which table the State fills out

**Comment by erinzstrenio** [comment]
@jgillis1 The content hasn't changed significantly during Sprint I. The changes I've made in the doc reflect what we're capable of building - it started as help drawer text then we decided against help drawers for now (not 508 compliant, not a reusable component for other sections, etc), so I reformatted the content to reflect these changes. I removed the checkboxes yesterday per my chat with Greg/Andrew, and reformatted the content to make sense without a checkbox. But the actual words haven't changed!
